synapse/federation/transport/client.py

```diff
@@ -181,6 +181,7 @@ def send_transaction(self, transaction, json_data_callback=None):
             long_retries=True,
             backoff_on_404=True,  # If we get a 404 the other side has gone
             try_trailing_slash_on_400=True,
+            retry_on_dns_fail=False,  # If we get DNS errors, the other side has gone
```

Comment:
For what its worth this was to fix a bug we had where our local DNS server would SERVFAIL fairly frequently

Reply:
@erikjohnston: which was? the addition of this line?